hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/ReadableFreezeUpgradeActions.java

```diff
@@ -221,11 +222,11 @@ private void extractAndReplaceArtifacts(
                 log.info("Finished generating config.txt and pem files into {}", artifactsLoc);
             }
             writeSecondMarker(marker, now);
-        } catch (final IOException e) {
+        } catch (final Throwable t) {
```

Comment:
What other exception beside IOException could be here? We should add those instead of Throwable?

Reply:
@JeffreyDallas found a case in the Solo environment where something other than `IOException` could be thrown...I don't remember what it was offhand. 🤔

Reply:
It was directory `data/upgrade/current` does not exist causes IllegalArgumentException from FileUtils

Reply:
Thanks @JeffreyDallas. I will change to that check later.